v2/tests/SugarL10n.spec.js

```diff
@@ -15,14 +15,18 @@ window.i18next = {
     getResourceBundle: jest.fn().mockReturnValue({}),
 };
 
+window.requirejs = jest.fn().mockImplementation((dependencies, callback) => {
```

Comment:
Hmmm. Are you really need to do that?
Is it specific to `i18next` or is it need for `webl10n` too?

Reply:
Yeah, it is needed if we are using require to load any library.